### PR TITLE
Add process.argv to retrieve username from command line arguments

### PR DESCRIPTION
**Pull request by Ilmaooo**

### CODE REVIEW

File: test/pageobjects/page.js [hunk 1]
```diff
@@ -21,4 +21,13 @@ export default class Page {
     await this.cartIcon.click()
   }
 
+  //get username from command line args
+  getUsernameFromArgs = () => {
```
Comment (Midjoni) [inline test/pageobjects/page.js]: Can this be removed?

File: test/specs/addToCartTest.smoke.js [hunk 2]
```diff
@@ -1,14 +1,18 @@
...
 import LoginPage from '../pageobjects/login.page.js'
 import HomePage from '../pageobjects/home.page.js'
 import ProductPage from '../pageobjects/product.page.js'
 import CartPage from '../pageobjects/cart.page.js'
 
+
 describe('Add to cart', () => {
   it('should add product to cart from product page', async () => {
+    
+    console.log('Username:', global.username)
```
Comment (Midjoni) [inline test/specs/addToCartTest.smoke.js]: No need to use `global` when referencing global vars

File: test/pageobjects/page.js (hunk 1 above)
Comment (Ilmaooo) [reply test/pageobjects/page.js]: done

File: test/specs/sortByPriceTest.regression.js [hunk 3]
```diff
@@ -1,10 +1,14 @@
 import HomePage from '../pageobjects/home.page.js'
 import LoginPage from '../pageobjects/login.page.js'
 
+
 const executeSortTest = async (sortOption) => {
+
+  console.log('Username:', username)
```
Comment (Midjoni) [inline test/specs/sortByPriceTest.regression.js:7]: In files where we reference global vars, it is good practice to make the first line of the file /* global var1, var2, var3 ... */

File: test/specs/addToCartTest.smoke.js (hunk 2 above)
Comment (Ilmaooo) [reply test/specs/addToCartTest.smoke.js]: removed

File: test/specs/sortByPriceTest.regression.js (hunk 3 above)
Comment (Ilmaooo) [reply test/specs/sortByPriceTest.regression.js:7]: done

File: test/specs/addToCartTest.smoke.js [hunk 4]
```diff
@@ -1,14 +1,18 @@
...
 
 describe('Add to cart', () => {
   it('should add product to cart from product page', async () => {
+    console.log('Username:', username)
+
     // Login and navigate to the home page
     await LoginPage.open()
-    await LoginPage.login(process.env.USERNAME1, process.env.PASSWORD)
+    await LoginPage.login(username, process.env.PASSWORD)
+    console.log('username:', username)
```
Comment (Midjoni) [inline test/specs/addToCartTest.smoke.js]: We can remove this log

**Review by Midjoni — APPROVED** [review]
LGTM

**Pull request merged**